app/controllers/concerns/hydra/collections/selects_collections.rb

```diff
@@ -11,16 +11,35 @@ def access_levels
     { read: [:read, :edit], edit: [:edit] }
   end
 
-  # add one of the following methods as a before filter on any page that shows the form_for_select_collection
+  ##
+  # Return list of collections for which the current user has read access.
+  # Add this or find_collections_with_edit_access as a before filter on any page that shows the form_for_select_collection
+  #
+  # @return [Array<SolrDocument>] Solr documents for each collection the user has read access
   def find_collections_with_read_access
     find_collections(:read)
   end
 
-  def find_collections_with_edit_access
+  ##
+  # Return list of collections for which the current user has edit access.  Optionally prepend with default
+  # that can be used in a select menu to instruct user to select a collection.
+  # Add this or find_collections_with_read_access as a before filter on any page that shows the form_for_select_collection
+  #
+  # @param [TrueClass|FalseClass] :include_default if true, prepends the default_option; otherwise, if false, returns only collections
+  # @param [Fixnum] :default_id for select menus, this is the id of the first selection representing the instructions
+  # @param [String] :default_title for select menus, this is the text displayed as the first item serving as instructions
+  #
+  # @return [Array<SolrDocument>] Solr documents for each collection the user has edit access, plus optional instructions
+  def find_collections_with_edit_access(include_default=false, default_id=-1, default_title="Select collection...")
     find_collections(:edit)
+    default_option = SolrDocument.new(id: default_id, title_tesim: default_title)
```

Comment:
What would you think of allowing folks to inject whatever they want as a default, rather than constructing a `SolrDocument` for them? Like the following:

```ruby
def find_collections_with_edit_access(default_option: nil)
  @user_collections.unshift(default_option) if default_option
  find_collections(:edit)
end

find_collections_with_edit_access(default: SolrDocument.new(id: -1, title_tesim: 'Select collection...'))
```

This :arrow_up: change also leaves `find_collections(:edit)` as the last line in the method which preserves its return value (which may or may not have been unintended on your part).